### PR TITLE
Create MaximiumElement.java

### PR DESCRIPTION
**Pull request by imshreya2000**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [x] All new code is formatted with `clang-format -i --style=file path/to/your/file.java`

### CODE REVIEW

**Review by susanhsrestha — APPROVED** [review]
Looks good to me!

**Comment by susanhsrestha** [comment]
Make correction to PR name

File: src/main/java/com/thealgorithms/divideandconquer/MaximimElement.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+public class DivideAndConquerMax {
```
Comment (susanhsrestha) [inline src/main/java/com/thealgorithms/divideandconquer/MaximimElement.java:1]: The package name should be included here.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

**Comment by imshreya2000** [comment]
Can you please approve and merge this PR

File: src/main/java/com/thealgorithms/divideandconquer/MaximimElement.java (hunk 1 above)
Comment (susanhsrestha) [inline src/main/java/com/thealgorithms/divideandconquer/MaximimElement.java:1]: I guess the packaging is required by the code maintainers.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

**Comment by imshreya2000** [comment]
Added the package name

**Comment by imshreya2000** [comment]
Made correction to PR name.

**Review by susanhsrestha — APPROVED** [review]
Looks good to me!

**Comment by imshreya2000** [comment]
Can you please merge this PR.

**Pull request closed without merging**